<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 7px; 
background-color: rgb(76, 76, 76);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="result">
      <property name="minimumSize">
       <size>
        <width>267</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200000</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(76, 76, 76);
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="delBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="reversBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="procentBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="divisBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(246, 147, 51);
color: rgb(255, 255, 255);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn8">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn9">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="multBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(246, 147, 51);
color: rgb(255, 255, 255);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btn6">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="minusBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(246, 147, 51);
color: rgb(255, 255, 255);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="plusBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(246, 147, 51);
color: rgb(255, 255, 255);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="btn0">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="commaBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(224, 224, 224);
color: rgb(0, 0, 0);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="equalsBtn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(246, 147, 51);
color: rgb(255, 255, 255);
margin: 3px;
}
QPushButton:pressed{
background-color:  rgb(118, 118, 118);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">/* Стилизация основного QComboBox */
QComboBox {

    border-radius: 7px;     /* Закругленные углы */
         /* Внутренний отступ */
    background-color: rgb(224, 224, 224); /* Светло-серый фон */
    color: rgb(0, 0, 0);    /* Черный цвет шрифта */
}

/* Убираем границу у выпадающей части */
QComboBox::drop-down {
    border: 0px;
}

/* Стилизация выпадающего списка QComboBox */
QComboBox QAbstractItemView {
    border: 1px solid #000; /* Черная граница */
    border-radius: 7px;     /* Закругленные углы */
    background-color: rgb(224, 224, 224); /* Светло-серый фон */
    color: rgb(0, 0, 0);    /* Черный цвет шрифта */
}

/* Стилизация полосы прокрутки внутри QComboBox */
QComboBox QAbstractItemView QScrollBar:vertical {
    border: 0px;           /* Без границ */
    background: rgb(240, 240, 240); /* Фон полосы прокрутки */
    width: 7px;           /* Ширина полосы прокрутки */
    margin: 2px 0 2px 0;   /* Отступы сверху и снизу */
    border-radius: 20px;    /* Закругление углов полосы прокрутки */
}

/* Стилизация ручки полосы прокрутки (thumb) */
QComboBox QAbstractItemView QScrollBar::handle:vertical {
    background: rgb(180, 180, 180); /* Цвет ручки */
    min-height: 20px;               /* Минимальная высота */
    border-radius: 20px;             /* Закругление углов ручки */
}

/* Кнопки вверх и вниз на полосе прокрутки */
QComboBox QAbstractItemView QScrollBar::add-line:vertical,
QComboBox QAbstractItemView QScrollBar::sub-line:vertical {
    background: rgb(224, 224, 224); /* Фон кнопок */
    border: 0px;                    /* Без границ */
    height: 10px;                   /* Высота кнопок */
    subcontrol-origin: margin;
    border-radius: 7px;             /* Закругление углов кнопок */
}

/* Убираем стрелки на кнопках прокрутки */
QComboBox QAbstractItemView QScrollBar::up-arrow:vertical,
QComboBox QAbstractItemView QScrollBar::down-arrow:vertical {
    border: none;
}

/* Убираем пространство (спэйсеры) вокруг кнопок прокрутки */
QComboBox QAbstractItemView QScrollBar::add-page:vertical,
QComboBox QAbstractItemView QScrollBar::sub-page:vertical {
    background: none;
}
</string>
      </property>
      <item>
       <property name="text">
        <string>Калькулятор</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Валюта</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Данные</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Длина</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ИМТ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Масса</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Объем</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Площадь</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Скидка</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Скорость</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Температура</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Финансы</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
